apps/ae_socket_connector/lib/test_accounts.ex

```diff
@@ -10,9 +14,13 @@ defmodule TestAccounts do
       101, 141, 158, 185, 198, 87, 190, 11, 15, 96, 80, 225, 138, 111, 252, 37, 59, 78>>
   end
 
-  def initiatorPubkey do
-    <<106, 184, 29, 213, 77, 73, 184, 77, 59, 65, 33, 156, 241, 78, 239, 173, 39, 2, 126, 254, 111, 28, 73, 150, 6,
-      150, 66, 20, 47, 81, 213, 154>>
+  def initiatorPubkeyEncoded do
+    # here you can return account in format "ak_26xYuZJn..."
+    pub_key =
```

Comment:
Why not represent the pub_key as a string here?

Reply:
Fixed!